eclass/git-r3.eclass

```diff
@@ -4,35 +4,31 @@
 # @ECLASS: git-r3.eclass
 # @MAINTAINER:
 # Michał Górny <mgorny@gentoo.org>
-# @SUPPORTED_EAPIS: 4 5 6 7
+# @SUPPORTED_EAPIS: 5 6 7 8
 # @BLURB: Eclass for fetching and unpacking git repositories.
 # @DESCRIPTION:
 # Third generation eclass for easing maintenance of live ebuilds using
 # git as remote repository.
 
-case "${EAPI:-0}" in
-	0|1|2|3)
-		die "Unsupported EAPI=${EAPI} (obsolete) for ${ECLASS}"
-		;;
-	4|5|6|7)
-		;;
-	*)
-		die "Unsupported EAPI=${EAPI} (unknown) for ${ECLASS}"
-		;;
+case ${EAPI:-0} in
+	5|6|7|8) ;;
+	*) die "${ECLASS}: EAPI ${EAPI:-0} not supported" ;;
 esac
 
 EXPORT_FUNCTIONS src_unpack
 
 if [[ ! ${_GIT_R3} ]]; then
 
-PROPERTIES+=" live"
+if [[ ${EAPI} != [567] ]]; then
+	PROPERTIES="live"
+else
+	PROPERTIES+=" live"
+fi
```

Comment:
I mean, I get it, but is this really such a big win? not that you should remove it, but it increases cyclomatic complexity purely for a syntactic sugar?

Reply:
It's mainly a reminder for removal of the old `+=` assignment later.

Reply:
Dropped.